.travis.yml

```diff
@@ -278,8 +288,13 @@ _helm_deploy: &helm_deploy
         # check pods
         pods_ready
 
+        if [[ ${TRAVIS_BUILD_STAGE_NAME} == "esc" ]]; then
```

Comment:
In this test, we stop and start the Alfresco Repository pod. When Repository starts again he temporarily closes the Elasticsearch index for few seconds while the Repository readiness probe is true so the test starts but the search returns a 500 error.

Reply:
This comment is really essential to understand why we're waiting for 30 seconds, so I suggest adding it to the file.